### PR TITLE
Fixes #665 Convert remainder of house-based queries to use house constants

### LINKED ISSUE #665: Convert remainder of house-based queries to use house constants.
Following from #637, a lot of the queries are still using integers for determining the house in question, rather than the sitewide constants.

### PR DESCRIPTION
**Pull request by antonionoca**
Replaced magic numbers with constants.
Updated SQL queries.
Verified that tests are passing.

### CODE REVIEW

File: www/includes/dbtypes.php [hunk 1]
```diff
@@ -164,6 +164,9 @@
 );
 */
 
+// epobject types
+define ('GLOSSARY_ITEM', 2);
```
Comment (dracos) [inline www/includes/dbtypes.php]: This could go right at the top of the file, where there is a comment about `eptype` that this could replace.

File: www/includes/easyparliament/member.php [hunk 2]
```diff
@@ -21,7 +21,7 @@ class MEMBER {
     public $extra_info = array();
     // Is this MP THEUSERS's MP?
     public $the_users_mp = false;
-    public $house_disp = 0; # Which house we should display this person in
+    public $house_disp = HOUSE_TYPE_ROYAL; # Which house we should display this person in
```
Comment (dracos) [inline www/includes/easyparliament/member.php]: Whilst I see the reason for this, this is actually an undefined value, rather than the royal house constant. It should probably not have a default here.

File: www/includes/easyparliament/member.php [hunk 3]
```diff
@@ -502,7 +502,7 @@ public function party_text($party = null) {
         }
     }
 
-    public function entered_house($house = 0) {
+    public function entered_house($house = HOUSE_TYPE_ROYAL) {
```
Comment (dracos) [inline www/includes/easyparliament/member.php]: Similarly, here (and `current_member` below) should really have `$house = null` or similar, royal should not be the default, this is a legacy issue.

**Comment by antonionoca** [comment]
Thanks for the feedback. These are good suggestions. I'll apply them.

File: www/includes/easyparliament/member.php (hunk 2 above)
Comment (antonionoca) [reply www/includes/easyparliament/member.php]: I'll revert this change back to setting house_disp to zero. I'm not confident to set it to undefined until I add unit tests, I'd better not introduce null pointer exceptions :)

File: www/includes/easyparliament/member.php (hunk 3 above)
Comment (antonionoca) [reply www/includes/easyparliament/member.php]: same here, I'll revert this change

**Comment by dracos** [comment]
Great, thanks very much! I've now merged this.
